src/style.css

```
@@ -1 +1,88 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 8px;
+  font-weight: 300;
+}
+
+.input-container {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input-container__input {
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+}
+
+.input-container__input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.input-container__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input-container__input--big {
+  font-family: Avenir-Heavy, sans-serif;
```

apply font-family for `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit
```